compiler/src/dotty/tools/dotc/reporting/messages.scala

```diff
@@ -2429,6 +2429,15 @@ class SynchronizedCallOnBoxedClass(stat: tpd.Tree)(using Context)
         |you intended."""
 }
 
+class ExtensionNullifiedByMember(method: Symbol, target: Symbol)(using Context)
+  extends Message(ExtensionNullifiedByMemberID):
+  def kind = MessageKind.PotentialIssue
+  def msg(using Context) = i"Suspicious extension ${hl(method.name.toString)} is already a member of ${hl(target.name.toString)}"
+  def explain(using Context) =
+    i"""Extension method ${hl(method.name.toString)} will never be selected
+       |because ${hl(target.name.toString)} already has a member with the same name.
+       |It can be called as a regular method, but should probably be defined that way."""
```

Comment:
"Suspicious" is a bit vague, but the explanation is good, so I would just replace the content of `msg` by `explain` here.